recipes/lux/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This license is already included in the PYPI source distribution. Please use that file so that the license is always up to date.